src/components/vue-form-builder.vue

```diff
@@ -648,9 +648,12 @@ export default {
       const excludedLabels = [""];
 
       const filtered = this.controls.filter((control) => {
-        return control.label
-          .toLowerCase()
-          .includes(this.filterQuery.toLowerCase());
+        let filter = this.filterQuery.toLowerCase();
+        let result = control.label.toLowerCase(filter).includes();
```

Comment:
The `toLowerCase` method doesn't accept parameters, maybe do you mean?
```suggestion
        let result = control.label.toLowerCase().includes(filter);
```